Tools/jit/_stencils.py

```diff
@@ -2,11 +2,15 @@
 
 import dataclasses
 import enum
-import sys
 import typing
 
 import _schema
 
+# Number of 32-bit words needed to store the bit mask of external symbols
+SYMBOL_MASK_SIZE: int = 4
```

Comment:
Also not a huge fan of this magic constant, since it'll almost certainly need to be updated in the future (and may even be platform-dependent). This *should* be the same as `math.ceil(len(known_symbols) / 32)` once `known_symbols` is populated, right?

Reply:
Yes, you are right. With time this changes (likely increases) and we need to keep updating this number. Also I agree, this could be platform dependent.

Reply:
Removed.